test/acceptance/list_test.exs

```diff
@@ -132,6 +132,92 @@ defmodule Acceptance.ListTest do
       assert as_html(markdown) == {:ok, html, messages}
     end
 
+    test "tight lists have no paragraph wrappers" do
+      markdown = """
+      - a
+      - b
+      """
+      html = """
+      <ul>
+      <li>a
+      </li>
+      <li>b
+      </li>
+      </ul>
+      """
+      messages = []
+
+      assert as_html(markdown) == {:ok, html, messages}
+    end
+
```

Comment:
We will need some more tests, but if you want to dev incrementally, very fine with me.